code/__DEFINES/layers.dm

```diff
@@ -4,11 +4,16 @@
 #define CLICKCATCHER_PLANE -99
 
 #define PLANE_SPACE -95
+#define PLANE_SPACE_RENDER_TARGET "PLANE_SPACE"
 #define PLANE_SPACE_PARALLAX -90
+#define PLANE_SPACE_PARALLAX_RENDER_TARGET "PLANE_SPACE_PARALLAX"
 
 #define FLOOR_PLANE -2
+#define FLOOR_PLANE_RENDER_TARGET "FLOOR_PLANE"
 #define GAME_PLANE -1
+#define GAME_PLANE_RENDER_TARGET "GAME_PLANE"
 #define BLACKNESS_PLANE 0 //To keep from conflicts with SEE_BLACKNESS internals
+#define BLACKNESS_PLANE_RENDER_TARGET "BLACKNESS_PLANE"
```

Comment:
Yeah not this, I meant making it automatic in the planemaster New based on the plane number. Something like
```
#define PLANE_RENDER_TARGET(plane) "*planemaster_[plane]"

/obj/screen/plane_master/New()
  render_target = PLANE_RENDER_TARGET(plane)

/something/else/someproc()
  render_source = PLANE_RENDER_TARGET(BLACKNESS_PLANE)
```

Reply:
I started implementing this and imo that some plane masters are rendering and some aren't makes this overly fiddly to work with. You have to know ahead of time what type a plane is before you use it anywhere no matter how I can think of implementing this, which is a lot less usable than the current implementation.

Reply:
Ah right you need the asterisk or not in different cases. Alright this is fine on an individual basis I guess.